testhelpers/unit_test_suite.go

```diff
@@ -29,10 +30,39 @@ func SetupTest() {
 }
 
 const (
-	TestSchemaName = "gftest"
-	TestTable1Name = "test_table_1"
+	TestSchemaName            = "gftest"
+	TestTable1Name            = "test_table_1"
+	TestCompressedTable1Name  = "test_compressed_table_1"
+	TestCompressedColumn1Name = "data"
+	TestCompressedData1       = "\x08" + "\x0cabcd" + "\x01\x02" // abcdcdcd
+	TestCompressedData2       = "\x08" + "\x0cabcd" + "\x01\x01" // abcddddd
 )
 
+var (
+	TestCompressedData3 string
+	TestCompressedData4 string
+)
+
+func init() {
+	filePaths := []string{
+		FixturePath("urls1.snappy"),
+		FixturePath("urls2.snappy"),
+	}
+
+	decompressed := [][]byte{}
+	for _, path := range filePaths {
```

Comment:
Nitpick, but this whole loop thing seems a bit unnecessary to me.. I'd create some kind of `LoadFixtureFromFile` helper function and then just do
```golang
func init() {
  TestCompressedData3 = LoadFixtureFromFile("urls1.snappy")
  TestCompressedData4 = LoadFixtureFromFile("urls2.snappy")
}
```

Reply:
Will update. I like this more 👍